helpers/regions.js

```diff
@@ -18,6 +18,7 @@ var regions = [
 module.exports = {
 	all: regions,
 	cloudfront: ['us-east-1'], // CloudFront uses the default global region
+	autoscaling: ['us-east-1'], // autoscaling uses the default global region
```

Comment:
I just confirmed - ASG should have all the regions. Doing the "describeAutoScalingGroups" call across all regions does work. Please change this to `autoscaling: regions,`.